src/main.zig

```diff
@@ -142,6 +145,50 @@ fn help(name: []const u8, args: *std.process.ArgIterator, allocator: std.mem.All
     try std.io.getStdOut().writer().print("\n", .{});
 }
 
+pub fn init(name: []const u8, args: *std.process.ArgIterator, allocator: std.mem.Allocator) !void {
+    _ = args;
+    _ = name;
+    _ = allocator; // defining these as _ to silence the compiler
+
+    var path_buffer: [std.fs.MAX_PATH_BYTES]u8 = undefined;
+    const main_path = std.fs.cwd().realpath("main.orng", &path_buffer) catch |err| switch (err) {
+        error.PathAlreadyExists => {
+            std.log.err("main.orng already present.", .{});
+            return error.BuildOrngError;
+        },
+        else => return err,
+    };
+
+    const build_path = std.fs.cwd().realpath("build.orng", &path_buffer) catch |err| switch (err) {
+        error.PathAlreadyExists => {
+            std.log.err("build.orng already present.", .{});
+            return error.BuildOrngError;
+        },
+        else => return err,
+    };
+
+    var main_orng = try std.fs.cwd().createFile(main_path, .{});
+    defer main_orng.close();
+
+    const main_content =
+        \\import std::debug;
+        \\fn main() void {
+        \\    debug::println("Hello, World!");
+        \\}
+    ;
+    try main_orng.writer().writeAll(main_content);
+
+    var build_orng = try std.fs.cwd().createFile(build_path, .{});
+    defer build_orng.close();
+
+    const build_content =
+        \\fn build() Package {
+        \\    Package::executable(.root="main.orng");\\
```

Comment:
This looks more like Zig code here in this string than Orng code. These contents have to match what was in the issue, so something like:
```zig
const main_content = 
    \\import std::debug
    \\
    \\fn main() -> () {
    \\    debug::println("Hello, World!")
    \\}
;

const build_content =
    \\fn build() -> Package {
    \\    Package::executable(.root="main.orng")
    \\}
;
```